api/tests/test_winery.py

```diff
@@ -179,15 +185,17 @@ def test_wine_detail_get(self):
 
     def test_varietal_get(self):
         response = self.client.get(
-            reverse('varietals')
+            reverse('varietals-list')
         )
-        expected = [{'id': k, 'value': v} for k, v in VARIETALS]
+        expected = [{'id': self.varietal.id, 'name': self.varietal.name}]
```

Comment:
We should keep the same keys so we don't break the current frontend implementation.